rustls/src/webpki/server_verifier.rs

```diff
@@ -122,6 +123,18 @@ pub struct WebPkiServerVerifier {
     supported: WebPkiSupportedAlgorithms,
 }
 
+impl fmt::Debug for WebPkiServerVerifier {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.debug_struct("WebPkiServerVerifier")
+            .field("roots", &format!("({} roots)", &self.roots.len()))
```

Comment:
Should we change this on the `RootCertStore` `Debug` impl instead of on the `WebPkiServerVerifier` impl? Seems more principled -- if the long list of roots is a problem for `WebPkiServerVerifier` it should equally be a problem for `WebPkiClientVerifier` and other types that contain a `RootCertStore`.

Reply:
Agreed, I think that would make the most sense.

Jsha mentioned:
>  it seemed useful to have some impl that actually would print the full contents of the RootCertStore

I recently added the [RootCertStore::subjects](https://github.com/rustls/rustls/blob/de41c70959844b5411c5018147a8a48c7817e387/rustls/src/webpki/anchors.rs#L80-L92) fn that returns `Vec<DistinguishedName>`. Since `DistinguishedName` impl's `Debug` that seems like a pretty good way to get this kind of output if you needed it.

Reply:
Sorry, missed that note.

Reply:
I'll move this to the `RootCertStore`. 👍🏻